package.json

```diff
@@ -64,6 +64,7 @@
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
     "hawk": "3.1.0",
+    "halfpenny": "*",
```

Comment:
version bumps to halfpenny in the short term are _very, very likely_.  we could tie it down to v0.0.1, which I'm ok with too!